vllm/multimodal/image.py

```diff
@@ -1,11 +1,11 @@
 from functools import lru_cache
-from typing import List, Optional, Tuple, TypeVar
+from typing import Any, List, Optional, Tuple, TypeVar
 
 import torch
 from PIL import Image
 from transformers import PreTrainedTokenizerBase
 
-from vllm.config import ModelConfig
+# from vllm.config import ModelConfig
```

Comment:
I found there's circular dependency issue with this. I will either fix it or revert it